<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>953</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>Lip-to-edge distance</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_2">
        <property name="text">
         <string>Angle of incidence</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_3">
        <property name="text">
         <string>Offset</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_4">
        <property name="text">
         <string>Flow</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_5">
        <property name="text">
         <string>Notes</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="statesFromNotesButton">
        <property name="text">
         <string>States from notes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addCorrectionButton">
        <property name="text">
         <string>Add Correction</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="scaleTimeButton">
        <property name="text">
         <string>Scale time</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="changeDurationButton">
        <property name="text">
         <string>Change duration</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="seeMotorRefsButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>See X, Z and Alpha</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="manualControlButton">
        <property name="text">
         <string>Manual Control</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QSlider" name="horizontalSlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="checkPathButton">
        <property name="text">
         <string>Check path feasibility</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="goToCoursorButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Go to coursor</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="playButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Play</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearPlotButton">
        <property name="text">
         <string>Clear Plot</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="recordCheckBox">
        <property name="text">
         <string>Record</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Composite</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="PlotWidget" name="graphicsView"/>
      </item>
      <item>
       <widget class="PlotWidget" name="graphicsView_2"/>
      </item>
      <item>
       <widget class="PlotWidget" name="graphicsView_3"/>
      </item>
      <item>
       <widget class="PlotWidget" name="graphicsView_4"/>
      </item>
      <item>
       <widget class="PlotWidget" name="graphicsView_5"/>
      </item>
     </layout>
    </item>
    <item row="7" column="0" rowspan="2">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" rowspan="2">
       <widget class="QPushButton" name="softStopButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>SOFT STOP</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>859</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpen_recent">
     <property name="title">
      <string>Open recent</string>
     </property>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="menuOpen_recent"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuPlot">
    <property name="title">
     <string>Plot</string>
    </property>
    <addaction name="actionLip_to_edge_distance"/>
    <addaction name="actionIncident_angle"/>
    <addaction name="actionOffset"/>
    <addaction name="actionPosition"/>
    <addaction name="actionMouth_Pressure"/>
    <addaction name="actionMass_Flow_Rate"/>
    <addaction name="actionVolume_Flow_Rate"/>
    <addaction name="actionAir_Temperature"/>
    <addaction name="actionSound_Frequency"/>
    <addaction name="actionX"/>
    <addaction name="actionZ"/>
    <addaction name="actionAlpha"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionChange_to_task_space"/>
    <addaction name="actionChange_to_joint_space"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuPlot"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actiondd">
   <property name="text">
    <string>dd</string>
   </property>
  </action>
  <action name="actionLip_to_edge_distance">
   <property name="text">
    <string>Lip-to-edge distance</string>
   </property>
  </action>
  <action name="actionIncident_angle">
   <property name="text">
    <string>Incident angle</string>
   </property>
  </action>
  <action name="actionOffset">
   <property name="text">
    <string>Offset</string>
   </property>
  </action>
  <action name="actionPosition">
   <property name="text">
    <string>Position</string>
   </property>
  </action>
  <action name="actionMouth_Pressure">
   <property name="text">
    <string>Mouth Pressure</string>
   </property>
  </action>
  <action name="actionMass_Flow_Rate">
   <property name="text">
    <string>Mass Flow Rate</string>
   </property>
  </action>
  <action name="actionVolume_Flow_Rate">
   <property name="text">
    <string>Volume Flow Rate</string>
   </property>
  </action>
  <action name="actionAir_Temperature">
   <property name="text">
    <string>Air Temperature</string>
   </property>
  </action>
  <action name="actionSound_Frequency">
   <property name="text">
    <string>Sound Frequency</string>
   </property>
  </action>
  <action name="actionX">
   <property name="text">
    <string>X</string>
   </property>
  </action>
  <action name="actionZ">
   <property name="text">
    <string>Z</string>
   </property>
  </action>
  <action name="actionAlpha">
   <property name="text">
    <string>Alpha</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionChange_to_task_space">
   <property name="text">
    <string>Change to task space</string>
   </property>
  </action>
  <action name="actionChange_to_joint_space">
   <property name="text">
    <string>Change to joint space</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
